R/dash.R

```diff
@@ -532,10 +545,7 @@ Dash <- R6::R6Class(
       ))
 
       # normalizes local paths and keeps newer versions of duplicates
```

Comment:
do we expect that there will be duplicates? we don't do this resolve step on the python side - I suppose that a component author could make a mistake by including a JS file with the same name twice when specifying that component's dependencies.

Reply:
This goes back to Nicolas's earlier example. I agree that it seems pretty far-fetched in practice, if we end up :hocho: references to `htmltools`, this will go away too.